certified-connectors/StarRez REST v1/apiDefinition.swagger.json

```diff
@@ -7691,14 +7691,84 @@
                   ]
                 },
                 "ETA": {
-                  "type": "string",
-                  "description": "Minutes since midnight for estimated time of arrival.",
-                  "title": "ETA"
+                  "type": "object",
```

Comment:
The type has been changed for the input field. Existing flows will continue to send the field with the old type. ValuePath: operation/SelectTermSession/parameter/body/schema/ETA/type 

Please confirm that the above change will not affect to the existing connection.

Reply:
I will revert this field back to string